.github/workflows/sync_docs.yml

```diff
@@ -60,6 +60,7 @@ jobs:
       - run: |
           rm -rf "pantsbuild.org/${{ steps.get-destination-dir.outputs.DESTINATION_DIR }}"
           mkdir "pantsbuild.org/${{ steps.get-destination-dir.outputs.DESTINATION_DIR }}"
+          cp help-all.json "pantsbuild.org/${{ steps.get-destination-dir.outputs.DESTINATION_DIR }}"
```

Comment:
I can't comment directly on the line, but the `pants help-all` invocation used in this script appears to be different to the one in the PR description; is that intentional?

Reply:
yeah; cool 👍